examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Hybrid_Jobs/Using_PennyLane_with_Braket_Hybrid_Jobs.ipynb

```diff
@@ -25,7 +25,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    print(job.result()["task summary"])
</pre><p>task summary is empty?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>